### ISSUE TITLE
Can't get to mining due to: No AMD OPENCL or NVIDIA CUDA GPU found

### ISSUE THREAD

**Issue by pinguweed** [nanopool/Claymore-Dual-Miner#134]
This is my start.bat file:

setx GPU_FORCE_64BIT_PTR 0
setx GPU_MAX_HEAP_SIZE 100
setx GPU_USE_SYNC_OBJECTS 1
setx GPU_MAX_ALLOC_PERCENT 100
setx GPU_SINGLE_ALLOC_PERCENT 100

EthDcrMiner64.exe -epool http://eth-eu.dwarfpool.com:80/0x91233E99B00D32856cEc24D396518c03E81f9149 -epsw x

This is the output i get:

C:\Users\akselforberg\Desktop\Claymore's Dual Ethereum+Decred_Siacoin_Lbry_Pascal AMD+NVIDIA GPU Miner v10.1>setx GPU_FORCE_64BIT_PTR 0

SUCCESS: Specified value was saved.

C:\Users\akselforberg\Desktop\Claymore's Dual Ethereum+Decred_Siacoin_Lbry_Pascal AMD+NVIDIA GPU Miner v10.1>setx GPU_MAX_HEAP_SIZE 100

SUCCESS: Specified value was saved.

C:\Users\akselforberg\Desktop\Claymore's Dual Ethereum+Decred_Siacoin_Lbry_Pascal AMD+NVIDIA GPU Miner v10.1>setx GPU_USE_SYNC_OBJECTS 1

SUCCESS: Specified value was saved.

C:\Users\akselforberg\Desktop\Claymore's Dual Ethereum+Decred_Siacoin_Lbry_Pascal AMD+NVIDIA GPU Miner v10.1>setx GPU_MAX_ALLOC_PERCENT 100

SUCCESS: Specified value was saved.

C:\Users\akselforberg\Desktop\Claymore's Dual Ethereum+Decred_Siacoin_Lbry_Pascal AMD+NVIDIA GPU Miner v10.1>setx GPU_SINGLE_ALLOC_PERCENT 100

SUCCESS: Specified value was saved.

C:\Users\akselforberg\Desktop\Claymore's Dual Ethereum+Decred_Siacoin_Lbry_Pascal AMD+NVIDIA GPU Miner v10.1>EthDcrMiner64.exe -epool http://eth-eu.dwarfpool.com:80/0x91233E99B00D32856cEc24D396518c03E81f9149 -epsw x

╔════════════════════════════════════════════════════════════════╗
║     Claymore's Dual ETH + DCR/SC/LBC/PASC GPU Miner v10.1      ║
╚════════════════════════════════════════════════════════════════╝

ETH: 1 pool is specified
Main Ethereum pool is eth-eu.dwarfpool.com:80
DCR: 0 pool is specified
AMD OpenCL platform not found
No NVIDIA CUDA GPUs detected.
No AMD OPENCL or NVIDIA CUDA GPUs found, exit

------
as you can see i have "No AMD OPENCL or NVIDIA CUDA GPUs found"
my graphicscard is intel hd 4600 and i have 8gb ram, windows 10
I am new to this stuff
I have downloaded Catalyst 2.0 (i think), but maybe thats the wrong version?

Thanks in advance to any helping souls
Aksel

**Comment by Roaders** [comment]
I am pretty sure that an Intel graphics card is not a Nvidia or an AMD graphics card...

**Comment by pinguweed** [comment]
Yes, I also raised an eyebrow to that, but as I couldnt find any info on what to do for intel graphics cards i assumed it wouldnt matter. What do i do then?

**Comment by jwb6658** [comment]
Not mine.

**Comment by pinguweed** [comment]
There is no way to mine with an intel graphics card? what makes the others so special?

**Comment by PatrickFranz** [comment]
LOL. What a helpful bunch!

**Comment by Samveda** [comment]
I m also facing the same problem
i m using 6 GPU mining rig with Nvidia GTX 1060 cards

On windows 7, I m getting this error. but when  i switch to windows 10, i m getting Error 12 (Not could enough resources)

I using latest nvidia driver and also tried with driver supplied with GPU cards but the problem is same

**Comment by alexgman** [comment]
Has there been any resolution to this issue?

**Comment by chai2** [comment]
getting the same when trying to run on ethos. Any update?

**Comment by Samveda** [comment]
I first installed all motherboard drivers n it solved my problem

**Comment by Samveda** [comment]
I hv only worked on Windows. Installed all motherboard drivers first

**Comment by juckuc** [comment]
Reinstalling the drivers helped in my case. As the nature suggests, the software does not see the graphic cards, and thus logical is to reinstall the drivers for the GPUs
